cmd/zq/zq.go

```diff
@@ -89,8 +89,8 @@ type Command struct {
 func New(f *flag.FlagSet) (charm.Command, error) {
 	cwd, _ := os.Getwd()
 	c := &Command{dt: resolver.NewTable()}
-	f.StringVar(&c.ifmt, "i", "auto", "format of input data [auto,ndjson,raw,zeek,zjson,zson]")
-	f.StringVar(&c.ofmt, "f", "zson", "format for output data [text,table,zeek,ndjson,raw,zson]")
+	f.StringVar(&c.ifmt, "i", "auto", "format of input data [auto,bzson,ndjson,zeek,zjson,zson]")
```

Comment:
The list at line 220 in this file needs to be adjusted similarly. (Maybe do a repo wide search for "raw" too...)

Reply:
good catch, thanks